assets/xml/overlays/ovl_Oceff_Spot.xml

```diff
@@ -0,0 +1,10 @@
+<Root>
+    <File Name="ovl_Oceff_Spot" BaseAddress="0x80972680" RangeStart="0x720" RangeEnd="0xDF8">
+        <Texture Name="sSunSongEffectTex" OutName="sun_song_effect" Format="i8" Width="32" Height="32" Offset="0x720"/>
+        <Array Name="sOceffSpotCylinderVtx" Count="27" Offset="0xB20">
+            <Vtx/>
+        </Array>
+        <DList Name="sOceffSpotCylinderMaterialDL" Offset="0xCD0"/>
+        <DList Name="sOceffSpotCylinderModelDL" Offset="0xD68"/>
```

Comment:
```suggestion
        <Array Name="sSunSongCylinderVtx" Count="27" Offset="0xB20">
            <Vtx/>
        </Array>
        <DList Name="sSunSongCylinderMaterialDL" Offset="0xCD0"/>
        <DList Name="sSunSongCylinderModelDL" Offset="0xD68"/>
```
I agree with Engineer here, all the names should probably be prefixed with the English name, rather than the actor name, especially since you've already done it for the texture. Note his would apply to all the `Oceff` PRs.